packages/stockflux-components/src/index.css

```diff
@@ -22,13 +22,45 @@
 
   --chart-teal: #37d8bc;
   --chart-teal-lighter: #51f2d6;
+  /* UX Colours */
+  /* Generic Colours */
+  --color-highlight: #29E9E3;
+  --color-inactive: #8A96A6;
+  --color-hover:#1B3B56;
+  --color-primary:#0D2437;
+  --color-secondary:#091A29;
+  /* Windows*/
+  --background-color:#0D2437;  
+  --header-bar:#071521;
+  --divider:#091A29;
+  --window-icon:#FFFFFF;
+  --stock-hover:#091A29;
```

Comment:
These could be prefixed with `window` and then the comments would be unnecessary, since the names will all begin with something describing what they are?

Reply:
Prefixed with `window-` on all of the variables. updated any references to them as needed.